pkg/providers/aws/credentials.go

```diff
@@ -62,6 +62,7 @@ var (
 				"ec2:DescribeRouteTables",
 				"ec2:CreateRoute",
 				"ec2:DeleteRoute",
+				"ec2:DescribeInstanceTypeOfferings",
```

Comment:
The permissions documentation needs to be updated with this new credential and @ciaran-byrne should be made aware of this update, as it'll be required for the next release.